### PR TITLE
add utils.filesystem.read_toml and depends on toml in moulinette

### PR DESCRIPTION
**Pull request by Psycojoker**
Pretty much straightforward

### CODE REVIEW

File: moulinette/utils/filesystem.py [hunk 1]
```diff
@@ -77,6 +80,28 @@ def read_yaml(file_path):
...
+    Safely read a toml file
+
+    Keyword argument:
+        file_path -- Path to the toml file
+    """
+
+    # Read file
+    file_content = read_file(file_path)
+
+    # Try to load toml to check if it's syntaxically correct
```
Comment (decentral1se) [inline moulinette/utils/filesystem.py]:
```suggestion
    # Try to load toml to check if it's syntactically correct
```

Unsure if "syntaxically" is a new cool way to say this though :)

File: moulinette/utils/filesystem.py [hunk 2]
```diff
@@ -77,6 +80,28 @@ def read_yaml(file_path):
...
+    try:
+        loaded_toml = toml.loads(file_content, _dict=OrderedDict)
+    except Exception as e:
+        raise MoulinetteError(errno.EINVAL,
+                              m18n.g('corrupted_toml',
```
Comment (decentral1se) [inline moulinette/utils/filesystem.py]: We're missing the `corrupted_toml` in the locales files?

File: moulinette/utils/filesystem.py (hunk 1 above)
Comment (kay0u) [reply moulinette/utils/filesystem.py]: In french we have "syntaxique" for "syntactic", like "connexion" for "connection" and a few others like that, it's very confusing --''.

File: moulinette/utils/filesystem.py (hunk 2 above)
Comment (Psycojoker) [reply moulinette/utils/filesystem.py]: Ah yes indeed '^'

**Comment by Psycojoker** [comment]
I realize that this PR is missing for the release, I'm surprise this isn't crashing the app code '^'

**Comment by alexAubin** [comment]
Yolomergin' ¯\\_(ツ)\_/¯

**Pull request merged**